application/backend/src/main/resources/apps/bulma/components/table/tablecell/dialog/.content.json

```diff
@@ -0,0 +1,30 @@
+{
+  "sling:resourceType": "wcm/dialogs/dialog",
+  "tabs": {
+    "sling:resourceType": "wcm/dialogs/components/tabs",
+    "generalTab": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "General",
+      "text": {
+        "sling:resourceType": "wcm/dialogs/components/richtext",
+        "name": "text",
+        "label": "Text",
+        "required": true
+      },
+      "colspan": {
+        "sling:resourceType": "wcm/dialogs/components/numberfield",
+        "name": "colspan",
+        "min": 1,
+        "max": 100,
+        "label": "Colspan"
```

Comment:
Authors can not understand colspan/rowspan as they're from HTML. Let's rename them to Columns and Rows.

Reply:
Fixed!